src/model/writer.js

```diff
@@ -802,14 +806,19 @@ export default class Writer {
 	 *
 	 *		addMarker( markerName, { range, usingOperation: true } );
 	 *
+	 * Create marker that doesn't affects the data model:
```

Comment:
I started commenting on this so I'll continue – we don't use contractions in the docs ("doesn't" should be changed to "does not").